### DISCUSSION TITLE
v3.0.0-rc.14

### DISCUSSION THREAD

**Discussion by pi0** [nuxt/nuxt#18393] [Announcements]
> **Note**
> This is the last release candidate for Nuxt v3! Are you ready?  👀

#### ⚠️  Breaking Changes

  - **cli:** Setup nuxt globally with `nuxt test` ([#4578](https://github.com/nuxt/framework/pull/4578))
  - **nuxt:** Only add `$f` fetch prefix to auto-keys ([#8852](https://github.com/nuxt/framework/pull/8852))
  - **test-utils:** Use `vitest/node` subpath export ([#8815](https://github.com/nuxt/framework/pull/8815))
  - **nuxt:** Remove `initialCache` option ([#8885](https://github.com/nuxt/framework/pull/8885))
  - **nuxt:** Enable payload extraction only for `nuxi generate` ([#9018](https://github.com/nuxt/framework/pull/9018))
  - **nuxt:** Include request url and params in `useFetch` key ([#6632](https://github.com/nuxt/framework/pull/6632))
  - **nuxt:** Fix typo for `NuxtRenderHTMLContext.bodyPrepend` ([#8712](https://github.com/nuxt/framework/pull/8712)) ([#8704](https://github.com/nuxt/framework/pull/8704))
  - **nuxt:** Remove support for `404.vue` shorthand ([#8809](https://github.com/nuxt/framework/pull/8809))
  - **kit:** Remove support for module container ([#9010](https://github.com/nuxt/framework/pull/9010))
  - **nuxt:** Move `head` option support into `defineNuxtComponent` ([#8901](https://github.com/nuxt/framework/pull/8901))
  - Remove deprecated api ([#9029](https://github.com/nuxt/framework/pull/9029))

[compare changes](https://github.com/nuxt/framework/compare/v3.0.0-rc.13...release/rc-14)

👉  **[Full Changelog](https://github.com/nuxt/framework/releases/tag/v3.0.0-rc.14)**

**Comment by Vi-jay** [comment]
Isn't the official version released today?

**Reply by kissu** [reply]
What do you mean? Nuxt getting out of RC and being 100% stable? 🎊 
It is still planned indeed. 😉 

Most VueJS' big announcements have (tested) "last minute" pushes so that it creates the surprise.
You even have commits 2 minutes before a conference's start sometimes. 😂 
But usually, those kind of releases are totally fine! 🎩

**Reply by gcentervn** [reply]
(*) Edge releases are automated after each commit passing tests
Read here: https://v3.nuxtjs.org/community/roadmap#current-releases

**Comment by kissu** [comment]
I speak on behalf of the community in thanking the team for their **huge daily** deliveries. 😻 
The amount of fixes + improvements in the last weeks/months was relentless and achieved with the collaboration of everybody here!
Thank you for giving of yourself as humans in every possible way! 🙏🏻 

See you all in a few hours at [NuxtNation](https://nuxtnation.com/) for the actual official announcements. 🎊

**Reply by alvarosabu** [reply]
@Maxp777  I'm sorry you feel like that tho, though maybe take a look at the commits involved in the Nuxt3 development, to see if validates your impression. 

About the error, have you checked if there is an open ticket already? If not you mind creating one with a reproducible so we can help you?

**Reply by kissu** [reply]
@Maxp777 people speaking at conferences and the ones making the project move forward code-wise are not always the same.
When they are, it's actually quite beneficial because it also provides some feedback for the core team, so that the final product can be improved. 💪🏻 
OSS is not only coding or promoting, it's also a lot of things in-between or mixed-up.

As a public speaker myself I can attest that this is necessary and actually...it doesn't take that much of your time (to prepare a talk) if you're doing something daily.
But don't think that Sébastien Chopin or Daniel Roe are spending their time creating fancy slides haha.

**Reply by BayBreezy** [reply]
Give it some time.. the team has been working real hard.
This is just the start of something great.. Nuxt v2 was not great when it was just released either but it matured into something beautiful. Nuxt v3 will follow the same path(all new tech stuff follow this path) its never perfect the first time but it does get better.

**Reply by kissu** [reply]
Not sure where you're getting your stats from, but [NPM does not agree with you at all](https://www.npmjs.com/package/nuxt).
On top of that, NPM download/such stats are quite naive overall and need to be taken into account sparingly.
There are quite a few reasons for such, won't get into details here about Vue2/Vue3 differences (already wrote [some leads in the comments of my answer](https://stackoverflow.com/a/74010696/8816585)) but you cannot make a direct comparison.

> Not count of commits imports.

Not sure what you mean by that one.

> All people find other solutions while waiting for nuxt 3.

The given solutions may be [Vitesse](https://github.com/antfu/vitesse), Astro or even [vite-plugin-ssr](https://github.com/brillout/vite-plugin-ssr)?
2 of those are tied pretty closely to our core team maintainers tbh. 😄  
Or maybe you're thinking about Laravel/else even? You can give Nuxt a try as of right now, may be totally fine for your use case. 😉 

If you have any frustration, there are people to help you. 💚 
They are here daily and only require a few minutes of your time (mainly a repro) to get everybody started on the same page.
You don't need to be angry at them, but at JS' pace at best.

If things break, it's because they are moving fast.
`Daniel Roe/Pooya/[...rest of our contributors]` are not breaking the currently working code just for fun. They are trying to move at the same pace + add features at the same time.

> No one will use some product because on TV people say that it is great. They will buy it once, will see that it is bad. And will drop it to trash.

I dare you to name me a single JS project that worked **flawlessly** out of the box in it's first iteration.
Please name at least one project that:
- caused 0 frustration
- got close to no breaking change 
- still bringing something quite new and deal breaker to the table

I can't think about any of them even tho I've starred almost [1000 projects](https://github.com/kissu?tab=stars) as of today.

Now, regarding the 📺 part. I don't feel like a conference is a TV or at least, a close medium in any way.
You are not forced to listen to it, you are not forced to agree to it. You _can_ join [the event](https://nuxtnation.com/) and get hyped tho!

It's literally like hanging with some friends around something that the whole group enjoys. 🍻 
But keep in mind that it doesn't block anything regarding the pacing of the code delivery. You will not have 3 months allocated for a presentation and the speakers of the Nuxt team will not give prio to a conference over some critical piece of code delivery.

Meanwhile, you **need** to also promote your thing.
Nuxt is free. You're not paying anything and the team does not sell you any kind of freemium.
You can even fork it and do whatever you want with it. How cool is that? 🎊 
Keep in mind that it's `founder 101` here: you need to share the word around, let people know that things are moving so that they can be included into the discussion, contribute to a feedback loop, get their issues/wishlist considered and so much more.

Basically: Community. 🤗 
All what Nuxt or any OSS is about, pretty much.
The core team is not selling a razor blade nor a VPN all over the place on TV, newspaper etc...but on a small platform with a few developers stocked for the latest goodies! 🍬 

> I think the team of people that know how to make nuxt is need to be bigger, to do things better.

Perfect transition. How do you grow a team?
By hiring more people. How do you feed them at the end of the month?
With money. How do you still make money as an OSS (Nuxt is free, as a reminder)?
By making your product nice and getting some investors to believe into your thing (startup fundraising pretty much).
Which is...pretty much one aspect as of why you get the conference in the first place.
So that people can see some progress. Get some money from investors. Feed their family. Grow together.

Does it sound fair viewed from that aspect? Do you see it as a "marketing" thing still?

> You can put thumbs down, but that not change situation, the impression from nuxt, it's magic disappeared.

I did not downvoted you, I respect your opinion.
As for the magic, as with everything in life: things tend to become boring/less exciting.
This is not a Nuxt thing, just how our human brain works: you get used to things.
It requires some work and some hype to move forward all together.

> May be because I previously not work with beta versions.
And in generally that all troubles with javascript and Node.js

That one could explain a lot indeed. 💯 
Welcome to the less exciting part of shipping a heavy update on a big product! 🚄 

---
To finish, if you want to get some positive vibes from the community.
This is [an excellent place](https://github.com/nuxt/framework/discussions/9064?sort=new) to fill yourself with positive energy and give Nuxt3 another try. 🙏🏻 
We're here to help you out in case something goes wrong. 💪🏻

**Comment by Vivi-Ornitier** [comment]
When I run `nuxi build` with rc.14, I get the following error.

```
 ERROR  Cannot read properties of undefined (reading 'options')                                                                                                            14:07:39

  at nuxtModule (node_modules/@nuxtjs/eslint-module/dist/module.js:25:13)
  at installModule (node_modules/nuxt/node_modules/@nuxt/kit/dist/index.mjs:416:9)
  at async initNuxt (node_modules/nuxt/dist/index.mjs:1827:7)
  at async loadNuxt (node_modules/nuxt/dist/index.mjs:1859:5)
  at async loadNuxt (node_modules/@nuxt/kit/dist/index.mjs:732:19)
  at async Object.invoke (node_modules/nuxi/dist/chunks/build.mjs:34:18)
  at async _main (node_modules/nuxi/dist/cli.mjs:50:20)
```

Do you know the reason?

**Reply by kissu** [reply]
Hi, do you mind sharing your currently used Node.js version?

**Reply by Vivi-Ornitier** [reply]
Hi,
I using Node.js v16.18.1.

**Reply by kissu** [reply]
Latest LTS for Node is `18.12.1`, mind giving that one at try? 😄

**Reply by BenjaminOddou** [reply]
Maybe this can be link to the deprecated options removed in this rc14 version ? See [this PR](https://github.com/nuxt/framework/pull/9029)

**Reply by Vivi-Ornitier** [reply]
It was caused by using @nuxtjs/eslint-module.
Thank you for answering.

**Reply by r-modica-cti** [reply]
@Vivi-Ornitier what did you do to resolve the issue? we are having the same one.

**Reply by alex-arabadjiev** [reply]
We are having the same issue with `@nuxt/typescript-build`. @Vivi-Ornitier could you tell us what you did to solve this issue?

**Comment by RajendraUppu** [comment]
i just updated my project from rc-12 to rc-14 i'm getting this weird error on v-model="modelValue" even tough  im emitting the modelValue... but in rc-12 it's working perfectly without any errors or warning.

it's appearing on codemirror component, from vue-codemirror library which supports two-way binding using modelValue.

[plugin:vite:vue] v-model cannot be used on a prop, because local prop bindings are not writable.
Use a v-bind binding combined with a v-on listener that emits update:x event instead.

**Reply by kissu** [reply]
Hi, do you mind sharing a repro for that one? 😄 
Seems like something unrelated to Nuxt so far.

**Reply by RajendraUppu** [reply]
Sure, ill update the repo link..

**Reply by RajendraUppu** [reply]
@kissu here you go: repo Link - https://stackblitz.com/edit/nuxt-starter-jobax1?file=app.vue

**Reply by RajendraUppu** [reply]
@kissu

**Reply by makhweeb** [reply]
@kissu

**Reply by CoreyAU** [reply]
@kissu Hi, this is still happening in the stable release. Can we please have some clarification if this is expected behaviour going forward or if it is a bug. Cheers

**Comment by oktapascal** [comment]
i just updated from rc 13 to rc 14. but i got error like this in nuxt.config, how to fix it?
<img width="657" alt="Screenshot 2022-11-16 133025" src="https://user-images.githubusercontent.com/53664617/202101725-b4d671dd-2aa4-458f-8230-062b97fb975b.png">

**Reply by sadeghbarati** [reply]
`buildModules` is deprecated in **nuxt3** use `nuxt.config.ts` (TypeScript) instead

[**use `modules`**](https://v3.nuxtjs.org/api/configuration/nuxt-config#modules)

**Reply by oktapascal** [reply]
Thanks for the advice

**Comment by iyashpal** [comment]
Can anyone help me to resolve this?

![image](https://user-images.githubusercontent.com/19922707/202104093-b8ae96ae-e936-499f-a992-c56797ed6a9d.png)

**Reply by RajendraUppu** [reply]
in nuxt.config.ts just rename autoImports to imports

**Reply by iyashpal** [reply]
```javascript
// https://v3.nuxtjs.org/api/configuration/nuxt.config
export default defineNuxtConfig({
    runtimeConfig: {
        SLUG: process.env.SLUG,
        
        APP_URL: process.env.APP_URL,
        API_URL: process.env.API_URL,
        
        MAIL_HOST: process.env.MAIL_HOST,
        MAIL_PORT: process.env.MAIL_PORT,
        MAIL_USERNAME: process.env.MAIL_USERNAME,
        MAIL_PASSWORD: process.env.MAIL_PASSWORD,
        MAIL_FROM_ADDRESS: process.env.MAIL_FROM_ADDRESS,

        public: {
            SLUG: process.env.SLUG,

            APP_URL: process.env.APP_URL,

            API_URL: process.env.API_URL,
        }

    },

    build: { transpile: ['@headlessui/vue'] },

    modules: [
        [
            '@pinia/nuxt',
            { imports: ['defineStore', ['defineStore', 'definePiniaStore']] },
        ],
        '@nuxt/image-edge',
        '@nuxtjs/tailwindcss',
    ],

    app: {
        head: {
            link: [
                { rel: 'stylesheet', href: 'https://rsms.me/inter/inter.css' }
            ],
        }
    },

    css: [
        '@/assets/scss/app.scss'
    ],

    image: {
        domains: ['*']
    }
})
```

Still the same.

**Reply by sadeghbarati** [reply]
There is no `addAutoImport` keyword in Nuxt repository

https://github.com/search?q=repo%3Anuxt%2Fframework+addAutoImport&type=code

Maybe you should clear your lockfiles

**Reply by sadeghbarati** [reply]
* Automated: `npx nuxi@latest upgrade --force`
* Manual: Bump `nuxt` dependency to `3.0.0-rc.14` and then use `npx nuxi@latest` cleanup to cleanup any local caches
* Clear your packageManager cache?

**Reply by iyashpal** [reply]
I have used `npx nuxi@latest upgrade --force` to upgrade.

**Reply by ExEr7um** [reply]
Same here. I have also used `npx nuxi@latest upgrade --force`, it didn't help.

**Reply by RajendraUppu** [reply]
Hello @iyashpal @ExEr7um 
i have tried replicating same with rc-14 and its working fine for me Repo Link : https://stackblitz.com/edit/nuxt-starter-kiusgr?file=nuxt.config.ts

and in nuxt.config.ts (https://pinia.vuejs.org/ssr/nuxt.html#auto-imports) it should be autoImports not imports as per the @pinia/nuxt docs.

modules: [
    // ...
    [
      '@pinia/nuxt',
      {
        autoImports: [
          // automatically imports `defineStore`
          'defineStore', // import { defineStore } from 'pinia'
          // automatically imports `defineStore` as `definePiniaStore`
          ['defineStore', 'definePiniaStore'], // import { defineStore as definePiniaStore } from 'pinia'
        ],
      },
    ],
  ],

some times npx nuxi@latest upgrade --force not helping try doing it manually and before doing it remove lock and node_modules. it worked for me.

**Reply by BenjaminOddou** [reply]
I am not sure that autoImports should be used as this option was deprecated 🧐. See [this PR](https://github.com/nuxt/framework/pull/9029)

**Reply by RajendraUppu** [reply]
@BenjaminOddou definitely agree, i think in rc-11 or 12 autoImports and defineNuxtconfig was deprecated.. but pinia needs this autoImports to work idk why pl check the [repo](https://stackblitz.com/edit/nuxt-starter-kiusgr?file=nuxt.config.ts) once.

**Reply by BenjaminOddou** [reply]
It's not needed only an option as you can manually import from pinia and use an alias. See [my personal portfolio](https://github.com/BenjaminOddou/portfolio). See nuxt.config.ts and piniaStore.ts in composables

**Reply by iyashpal** [reply]
@RajendraUppu I tried with both `autoImports` and `imports`. If I rollback to rc-13 then everything works well.

**Reply by ExEr7um** [reply]
Well I don't have neither `autoImports` nor `imports` in my config, but it still doesn't work. I rolled back to `rc.13` and everything works perfectly fine.

**Reply by BenjaminOddou** [reply]
Would you give a try at [this](https://github.com/nuxt/framework/issues/6623#issuecomment-1248418063)? 🧐

EDIT: alternatively [this solution](https://github.com/nuxt/framework/issues/6623#issuecomment-1262009146)

**Reply by ExEr7um** [reply]
I don't think it's an issue with Pinia. We both have `@nuxtjs/tailwindcss` and `@nuxt/image-edge`, will try to test without them.

**Reply by ExEr7um** [reply]
Well yes, it's an issue with `@nuxt/image-edge`. Made a reproduction: https://stackblitz.com/edit/nuxt-starter-idfvjn.

**Reply by wallchr** [reply]
Removing `@nuxt/image-edge` fixed the issue for me.

**Reply by iyashpal** [reply]
removing `@nuxt/image-edge` fixed the issue for me too.

**Reply by sadeghbarati** [reply]
https://github.com/nuxt/image/pull/648

https://github.com/nuxt/image/blob/4b27db34afc3e3e9aa58ca64275d245b6aa2fa79/src/module.ts#L2

**Reply by ChuanfengLai** [reply]
@pinia/nuxt@0.4.3 has solved this problem.

**Comment by dodanex** [comment]
Any ideas about this error? 
I have searched for `useMeta` inside my project and I can't find anything.

![Screenshot 2022-11-16 103803](https://user-images.githubusercontent.com/16241982/202130344-e019ca9c-8517-4eb5-bed9-9a2e174939af.png)

**Reply by harlan-zw** [reply]
`useMeta` was dropped in this PR https://github.com/nuxt/framework/pull/8901

Not sure why it would be detected, try deleting the .nuxt folder. otherwise can you list your modules

**Reply by wallchr** [reply]
https://github.com/vueuse/vueuse/pull/2449

**Reply by harlan-zw** [reply]
> [vueuse/vueuse#2449](https://github.com/vueuse/vueuse/pull/2449)

Nice catch! Will try get that pushed through

**Reply by sadeghbarati** [reply]
@harlan-zw 

since `vueuse/motion` use latest `@nuxt/kit": "npm:@nuxt/kit-edge@latest` that might caused error too

https://github.com/vueuse/motion/blob/main/src/nuxt/module.ts


---


<del>addAutoImport</del> => addImports

* [vueuse/motion](https://github.com/vueuse/motion) 
* [nuxt-modules/algolia](https://github.com/nuxt-modules/algolia)
* [github-module](https://github.com/nuxtlabs/github-module)
* [design-tokens](https://github.com/nuxt-modules/design-tokens)

**Reply by dodanex** [reply]
I think I found the issue: The problem was with `vueuse` config.

I had this inside `nuxt.config.ts`:

```js
vueuse: {
	ssrHandlers: true
}
```

Then after I commented `// ssrHandlers: true` it's working!
Thank you all!

**Reply by mittci** [reply]
Rename useMeta to useHead https://github.com/nuxt/framework/issues/4062

Is need to replace `useMeta` with `import { useHead } from '#imports'` ?!

**Comment by Vivi-Ornitier** [comment]
Removed initialCache option from useFetch?
TS2345 error started from rc.14.

**Reply by BayBreezy** [reply]
> Removed initialCache option from useFetch? TS2345 error started from rc.14.

Yup.. That was removed

**Reply by Vivi-Ornitier** [reply]
Are there alternative options?

**Reply by TheAlexLichter** [reply]
Nope. https://github.com/nuxt/framework/pull/8885

**Comment by abdonrd** [comment]
After update from `3.0.0-rc.13` to `3.0.0-rc.14`, we have this error:

```
vue-router.mjs:3451 Error: [🍍]: getActivePinia was called with no active Pinia. Did you forget to install pinia?
	const pinia = createPinia()
	app.use(pinia)
This will fail in production.
    at useStore (pinia.mjs:1696:19)
    at auth.ts:9:3
    at fn (nuxt.mjs?v=184e88fe:141:27)
    at callWithNuxt (nuxt.mjs?v=184e88fe:146:12)
    at router.mjs?v=184e88fe:106:1
```

**Reply by Amirali-Yavari** [reply]
please update your pinia

**Reply by abdonrd** [reply]
We are already on the latest version: `@pinia/nuxt@0.4.3` and [`pinia@2.0.23`](https://github.com/vuejs/pinia/releases/tag/pinia%402.0.23)

**Reply by BenjaminOddou** [reply]
you don't need pinia with pinia/nuxt. maybe try to uninstall both then do a : `npm install pinia @pinia/nuxt --legacy-peer-deps` then remove pinia with `npm uninstall pinia`

**Reply by abdonrd** [reply]
We install `pinia` because we also use `storeToRefs`:
```typescript
import { storeToRefs } from 'pinia';
```

**Reply by abdonrd** [reply]
And if we just rollback to the `nuxt@3.0.0-rc.13` everything works again.

**Reply by dodanex** [reply]
> We install `pinia` because we also use `storeToRefs`:
> 
> ```ts
> import { storeToRefs } from 'pinia';
> ```

Hope I'm not wrong but I think you already have `pinia` installed in your `nuxt_modules` because of `@pinia/nuxt`, so you can remove `pinia` as a dependency from your package.json and still be able to:
```js
import { storeToRefs } from 'pinia';
```

**Reply by abdonrd** [reply]
Even deleting pinia as dependency, still getting the same error.

**Reply by BenjaminOddou** [reply]
@abdonrd I am personally using @pinia/nuxt in my [portfolio](https://github.com/BenjaminOddou/portfolio). I am using only @pinia/nuxt and using an alias in the nuxt.config.ts file. For pinia stores, check the piniaStore.ts file under the composables folder.

All of this using the stable version of Nuxt 3.0.0

**Reply by abdonrd** [reply]
Still not working after [`@pinia/nuxt@0.4.4`](https://github.com/vuejs/pinia/blob/v2/packages/nuxt/CHANGELOG.md#044-2022-11-17)

**Reply by brunokunace** [reply]
same here, have an error ` getActivePinia was called with no active Pinia. Did you forget to install pinia? ` with this version 0.4.4

**Reply by mittci** [reply]
@abdonrd, @brunokunace, @BenjaminOddou for what is `  alias: { pinia: '/node_modules/@pinia/nuxt/node_modules/pinia/dist/pinia.mjs', },`?
You use some awkward module? 
I last seen that error in rc.9.

Also try to put `'@pinia/nuxt'` module last in order, but your last module is ` 'nuxt-schema-org'`, may be will help.
I have impression that modules are loaded in reverse order, first in stack first is loaded. Or nuxt-schema-org not need pinia?

**Reply by BenjaminOddou** [reply]
@Maxp777 @pinia/nuxt module contains pinia by itself. The problem is that

import { defineStore } from 'pinia'

point to '/node_modules/pinia/dist/pinia.mjs'.

That's why we need to use an alias to point to the right pinia location

**Reply by mittci** [reply]
Two pinia, double effect :) 
So if I put alias I can remove pinia from package.json (or I will have troubles like you)

**Reply by BenjaminOddou** [reply]
I don't have any troubles with pinia (check my [portfolio](https://www.benjaminoddou-photographe.com)) 😉. And yes the best is to remove pinia from package.json IMHO


[repository](https://github.com/BenjaminOddou/portfolio) of my portfolio

**Reply by abdonrd** [reply]
I just see that [@pinia/nuxt@0.4.1](https://github.com/vuejs/pinia/blob/v2/packages/nuxt/CHANGELOG.md#041-2022-08-18) change `buildModules` to `modules`.

Updated and now it's working for us!

**Reply by mittci** [reply]
https://nuxt.com/docs/migration/configuration/#migration-1
a ha ha ha buildModules is deprecated in nuxt 3, but you put it here

**Comment by Vivi-Ornitier** [comment]
I get the below error when I run the test. `$hello` is plugin.
It didn't happen in rc.13, did anything change in rc.14?

```
TypeError: _ctx.$hello is not a function
```

.vue
```
<template>
  <div>{{ $hello() }}</div>
</template>
```

**Reply by TheAlexLichter** [reply]
Hey! Could you kindly create an issue and provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Comment by vedmant** [comment]
I see that 'autoImports' was removed, how do I disable auto imports now? I have custom setup where auto imports are not used as it causes inconsistent code, so there is no need to spend any resource generating this auto import files.

**Comment by noelantoniooo** [comment]


**Reply by harlan-zw** [reply]
Use v3.0.0, try removing the module `vuero`

**Reply by harlan-zw** [reply]
This is not the correct code for a module, check how other modules use it

**Reply by noelantoniooo** [reply]
Thank you @harlan-zw

**Comment by hjleee93** [comment]
I got this error after update from rc11 to rc14
I deleted all node_modules and yarn.lock, still i got this error.


![스크린샷 2022-11-25 오후 5 11 02](https://user-images.githubusercontent.com/71309720/203932302-294301fb-5538-4f7c-a1ba-c9ce30b133f9.png)


`Cannot start nuxt:  this.addPlugin is not a function                                                                                                                                                                                                    17:07:03

  at module (node_modules/cookie-universal-nuxt/lib/module.js:10:8)
  at installModule (node_modules/nuxt/node_modules/@nuxt/kit/dist/index.mjs:416:9)
  at async initNuxt (node_modules/nuxt/dist/index.mjs:1827:7)
  at async load (node_modules/nuxi/dist/chunks/dev.mjs:6779:9)
  at async Object.invoke (node_modules/nuxi/dist/chunks/dev.mjs:6840:5)
  at async _main (node_modules/nuxi/dist/cli.mjs:50:20)`

**Reply by TheAlexLichter** [reply]
That's on the author of `cookie-universal-nuxt` to fix ☺️
